Analysis/include/EventInfo.h

```diff
@@ -241,15 +241,15 @@ class EventInfoBase {
 
 private:
     template<typename LorentzVector>
-    static bool PassEcalNoiceVetoJets(const LorentzVector& jet_p4, Period period)
+    static bool PassEcalNoiceVetoJets(const LorentzVector& jet_p4, Period period, bool jets_pu_id)
```

Comment:
`bool jets_pu_id` -> `int jet_pu_id`. So the condition below `jets_pu_id & 2 == 0` was always `true` in your  pu studies...